src/Extensions/ExtensionManager.php

```diff
@@ -13,18 +14,34 @@ public function getPackageManager()
 
     public function install($extensionId, $version = null)
     {
+        // @TODO temporary work around to increase memory
+        ini_set('memory_limit', '2gb');
```

Comment:
Yep, that would be great.

Might cause issues if more than one extension decides to manage RAM...